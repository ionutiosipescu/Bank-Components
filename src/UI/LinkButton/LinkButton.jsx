import React from "react";
import { Btn } from "../Button/Button.style";
import { Link } from "react-router-dom";

function LinkButton({ label, to, size, primary, handleClick, children }) {
  return (
    <Btn as={Link} to={to} size={size} onClick={handleClick} primary={primary}>
      {label}
      {children}
    </Btn>
  );
}

export default LinkButton;
