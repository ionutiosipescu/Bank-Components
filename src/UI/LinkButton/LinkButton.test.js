import React from "react";
import { shallow, mount, render } from "enzyme";
import LinkButton from "./LinkButton";

it("expect to render LinkButton component", () => {
  expect(shallow(<LinkButton />)).toMatchSnapshot();
});
