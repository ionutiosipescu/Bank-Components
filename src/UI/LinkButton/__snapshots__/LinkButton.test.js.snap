// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render LinkButton component 1`] = `
<styled.button
  as={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "render": [Function],
    }
  }
/>
`;
